### PR TITLE
fix: Use the correct message to print the current k8s context

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatoliy Bazko <abazko@redhat.com>

### What does this PR do?
Fix the message to print current k8s context
```bash
$ ./run workspace:list
› Current kuberntetes context: 'minikube'
Id                        Name Namespace Status  Created                  Updated  
...
```

### CODE REVIEW

File: src/api/kube.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export class KubeHelper {
   static initializeKubeConfig(): KubeConfig {
     const kc = new KubeConfig()
     kc.loadFromDefault()
-    cli.info(`Set current context to '${kc.currentContext}'`)
+    cli.info(`› Current kuberntetes context: '${kc.currentContext}'`)
```
Comment (AndrienkoAleksandr) [inline src/api/kube.ts:35]: kuberntetes -> kubernetes . And maybe from capital letter.

File: yarn.lock [hunk 2]
```diff
@@ -1525,7 +1525,7 @@ ecc-jsbn@~0.1.1:
 
 "eclipse-che@git://github.com/eclipse/che#master":
   version "0.0.0"
-  resolved "git://github.com/eclipse/che#bf62877c12ae1cbc0174bd42a683e6151a46d48e"
+  resolved "git://github.com/eclipse/che#2241525c799832c5b30d957603813565bcc539c0"
```
Comment (mmorhun) [inline yarn.lock:1528]: This change is not necessary.

File: src/api/kube.ts (hunk 1 above)
Comment (tolusha) [reply src/api/kube.ts:35]:
```suggestion
    cli.info(`› Current Kubernetes context: '${kc.currentContext}'`)
```

File: yarn.lock (hunk 2 above)
Comment (tolusha) [reply yarn.lock:1528]:
```suggestion
  resolved "git://github.com/eclipse/che#bf62877c12ae1cbc0174bd42a683e6151a46d48e"
```

**Pull request merged**